code/edgebridge/build.gradle

```diff
@@ -217,9 +218,12 @@ dependencies {
     testImplementation 'org.powermock:powermock-module-junit4:2.0.0'
 
     // Adobe Mobile dependencies for testing
-    implementation('com.adobe.marketing.mobile:edge:1+'){
-        exclude group: 'com.adobe.marketing.mobile', module: 'core'
-    }
+//    implementation('com.adobe.marketing.mobile:edge:1+'){
+//        exclude group: 'com.adobe.marketing.mobile', module: 'core'
+//    }
+    // TODO: replace these with the actual 2.0 Edge dependency when available
+    implementation fileTree(dir: 'libs', include: ['edge-release-2.0.0.aar', 'edgeidentity-release-2.0.0.aar'])
```

Comment:
are these dependencies used as for androidTests? if so use androidTestImplementation instead